exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -12,19 +12,76 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::ops::Range;
+use std::cmp;
```

Comment:
Please change the imports order: third-party first, then std, then inner usings.